src/imports/gsettings/qmlgsettingsschema.h

```diff
@@ -33,8 +33,8 @@ class QmlGSettingsSchema : public QObject
 {
     Q_OBJECT
     Q_PROPERTY(bool valid READ isValid NOTIFY validChanged)
-    Q_PROPERTY(QString id READ id WRITE setId)
-    Q_PROPERTY(QString path READ path WRITE setPath)
+    Q_PROPERTY(QString id MEMBER m_schemaId WRITE setId NOTIFY idChanged)
+    Q_PROPERTY(QString path MEMBER m_path WRITE setPath NOTIFY pathChanged)
```

Comment:
If it is member you should not need to emit it.

https://doc.qt.io/qt-5/properties.html
`A NOTIFY signal is optional. If defined, it should specify one existing signal in that class that is emitted whenever the value of the property changes. NOTIFY signals for MEMBER variables must take zero or one parameter, which must be of the same type as the property. The parameter will take the new value of the property. The NOTIFY signal should only be emitted when the property has really been changed, to avoid bindings being unnecessarily re-evaluated in QML, for example. Qt emits automatically that signal when needed for MEMBER properties that do not have an explicit setter.`

Reply:
Could you also return to READ id, READ path? The behavior should be identical in both cases, but accessor functions are more flexible and I see no reason to replace them with member associations.

Reply:
I have reverted these.